src/pygeoprocessing/routing/routing.pyx

```diff
@@ -696,6 +700,13 @@ def fill_pits(
     # have already been processed
     cdef int feature_id
 
+    # used to handle the case for single outlet mode
+

```

Comment:
remove empty line to better relate the comment to the code?

Reply:
removed in 5aaefd9